scripts/choco/tools/chocolateyinstall.ps1

```diff
@@ -5,17 +5,11 @@ $url64      = "https://github.com/kubeshop/tracetest/releases/download/v${env:ch
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
   unzipLocation = $toolsDir
-  fileType      = 'exe'
-  url64bit      = $url64
-
-  softwareName  = 'tracetest*'
```

Comment:
From chocolatey bot:

- There are no need for fileType or softwareName inside the $packageArgs since it is an archive extracted with Install-ChocolateyZipPackage, not an installer run with Install-ChocolateyPackage